packages/web-components/src/components/content-block-headlines/content-block-headlines-item.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { css, customElement, html } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import DDSContentItem from '../content-item/content-item';
+import styles from './content-block-headlines.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Content Block Headlines item
+ *
+ * @element dds-content-block-headlines-item
+ */
+@customElement(`${ddsPrefix}-content-block-headlines-item`)
+class DDSContentBlockHeadlinesItem extends StableSelectorMixin(DDSContentItem) {
+  render() {
```

Comment:
In this way our users won't have to add `slot="content"` to `<dds-content-block-headlines-item>`:

```suggestion
  /**
   * The shadow slot this headlines item should be in.
   */
  @property({ reflect: true })
  slot = 'content';

  render() {
```

Reply:
Ah, that's what I was missing, thanks!